README.md

```diff
@@ -1,6 +1,209 @@
+  </ol>
+</details>
 
-The thesis topic was provided by the Advanced Concepts Team (ACT) at The European Space Agency (ESA).  
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
+
+Trajectory Optimization for a Spacecraft Swarm (TOSS) is a thesis project presented for the degree of Master of Science in mathematics, with specialization in Optimization and Systems theory provided by KTH Royal Institute of Technology. The thesis topic was defined within the scope of $\Phi$[-lab@Sweden](https://www.ai.se/en/data-factory/f-lab-sweden) in the frame of a collaboration between [AI Sweden](https://www.ai.se/en/) and the [European Space Agency](https://www.esa.int/) to explore distributed edge learning for space applications. The project was mainly supervised by Pablo Gómez and Emmanuel Blazquez from the Advanced Concepts Team (ACT) at ESA, and examined by Xiaoming Hu from KTH.  
+
+The goal of the project is to enable an evaluation of the effects for single and multi-spacecraft architectures when applied to a prospective mission with specified science objectives. In particular, TOSS is designed to be:
+
+- **open-source**: the source code of TOSS is publicly available.
+- **decentralised**: each function and capability within the module can be adopted to generate trajectories for a wider variety of objectives and celestial bodies.
+- **user-expandable**: the user can expand the capabilities of the code using the decentralised structure. 
+
+The optimization is based on an evolutionary approach sampling a large population where each candidate solution results in a deterministics trajectory that is obtained by numerical intergration. The trajectories are then resamples for a desired fixed time-step and then evaluating according to a defined fitness function. Then, unless the algorithm has met a certain stopping criteria, the population is mutated accoridng to the best candidate solution and the optimisation continuous by reiterating the process again.
+
+<p align="center">
+  <a href="https://github.com/rasmusmarak/TOSS">
+    <img src="docs/source/TOSS_Optimization_Scheme.pdf" alt="Scheme"  width="910" height="459">
+  </a>
+  <p align="center">
+    Trajectory Optimisation of a Spacecraft Swarm
+    <br />
+    <a href="https://github.com/rasmusmarak/TOSS/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/rasmusmarak/TOSS/issues">Request Feature</a>
+  </p>
+</p>
+
+
+
+### Built With
+
+This project is based on:
+
+- [polyhedral-gravity-model](https://github.com/esa/polyhedral-gravity-model) a fast, parallelised version of the polyhedral gravity model implemented in C++ along with a python interface.
+- [DESolver](https://github.com/Microno95/desolver) a python library fro solving Initial Value Problems using numerical integrators. 
+- [PyGMo](https://esa.github.io/pygmo2/index.html) a scientific python library for massively parallel optimization managed by the Advanced Concepts Team at the European Space Agency
```

Comment:
pygmo is C++ with a python wrapper I think

Reply:
Fixed!